rules-reviewed/jboss-eap/websphere/WebSphere.windup.xml

```diff
@@ -0,0 +1,1219 @@
+            </perform>
+        </rule>
+        <rule id="WebLogic-weblogic.net.http.HttpsURLConnection">
+            <when>
+                <javaclass references="weblogic.net.http.HttpsURLConnection" as="default">
+                </javaclass>
+            </when>
+            <perform>
+                <hint message="Replace with `javax.net.ssl.HttpsURLConnection`" effort="2"/>
+            </perform>
+        </rule>
+
+        <!-- asynchbeans -->
+        <rule id="WebSphere>eap:asynchbeans.Work-6bcvk">
+            <when>
+                <javaclass references="com.ibm.websphere.asynchbeans.Work" as="default">
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <classification classification="WebSphere Asynchronous Work" effort="2">
+                    <link rel="sourceJavadoc" description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/api/content/nl/en-us/SS7JFU_7.0.0/com.ibm.websphere.javadoc.doc/web/apidocs/com/ibm/websphere/asynchbeans/Work.html"/>
+                </classification>
+                <hint message="`Work` is implemented by applications when they want to run code blocks asynchronously."/>
+                <!-- http://stackoverflow.com/questions/9026516/replacing-webspheres-workmanager-in-jboss -->
+                <hint>
+                    Depending on how what asyncbeans WorkManager is used for, it can be migrated to one of:
+                     * [Java EE 7: JSR-236](https://jcp.org/en/jsr/detail?id=236) - see [Migrate WebLogic Applications That Use the CommonJ Framework or Work Manager API to Red Hat JBoss Enterprise Application Platform 6](https://access.redhat.com/articles/1326763)
+                     * [EJB 3.0](http://jcp.org/en/jsr/detail?id=220) Timer Service
+                     * [EJB 3.1](http://jcp.org/en/jsr/detail?id=318) Asynchronous Beans
+                     * [Servlet 3.0](http://jcp.org/en/jsr/detail?id=315) Asynchronous Servlets
+
+                     <link description="Java EE 7: JSR-236" href="https://jcp.org/en/jsr/detail?id=236"/>
+                     <link description="EJB 3.0 Timer Service" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnboy.html"/>
+                     <link description="EJB 3.1 Asynchronous Beans" href="https://docs.oracle.com/javaee/6/tutorial/doc/gkkqg.html"/>
+                     <link description="Servlet 3.0 Asynchronous Servlets" href="https://blogs.oracle.com/enterprisetechtips/entry/asynchronous_support_in_servlet_3"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="WebSphere>eap6:-Cbxnb">
+            <when>
+                <javaclass references="com.ibm.websphere.asynchbeans.WorkManager" as="default"/>
+            </when>
+            <perform>
+                <hint message="Migrate to JBoss JCA Work Manager." effort="1">
```

Comment:
Is it enough to give a link to JCA Work Manager? Shouldn't we link to articles that tell you how to migrate?